documentation/bolt_installing_modules.md

```diff
@@ -131,18 +130,25 @@ these core modules.
     Collect facts for the specified targets from the configured PuppetDB
     connection and stores the collected facts on the targets.
 
-> 🔩 **Tip:** To override a packaged module with another version, download the
-> version you want and configure your module path to point to it.
-
 ### Modules that contain helper code for writing your own tasks
 
 -   [ruby_task_helper](https://forge.puppet.com/puppetlabs/ruby_task_helper): A
     helper for writing tasks in Ruby.
 -   [python_task_helper](https://forge.puppet.com/puppetlabs/python_task_helper):
     A helper for writing tasks in Python.
 
-> 🔩 **Tip:** To override a packaged module with another version, download the
-> version you want and configure your module path to point to it.
+## Overriding a packaged module
+
+To override a packaged module with another version, download the
+version you need and configure your module path to point to it.
+
+1. Create a directory for your module. For example, `module-override`.
+2. Download the version of the module you need and place it in the `module-override` directory.
```

Comment:
Users shouldn't have to modify the modulepath to override a packaged module, their own modulepath is already in front of the packaged one. Just installing the module from their Puppetfile into `modules` will override the packaged one.

Reply:
Okay cool. I'll update with your suggestion